measure/endurance-snapshot

```diff
@@ -475,8 +467,14 @@ fi
 echo "- Disk usage"
 LC_ALL=C df -k -P > $path/df
 
-echo "- statefs"
-dump_statefs_value namespaces/Screen/Blanked
+echo "- Blanked state"
+blanked_state=$(qdbus --system com.nokia.mce /com/nokia/mce/request com.nokia.mce.request.get_display_status)
+if [ "$blanked_state" == "on" ]; then
+	blanked_state="0"
+else
+	blanked_state="1"
+fi
+echo $blanked_state >> $path/blanked
```

Comment:
How about "$path/display_blanked"? Also can you avoid creating the file if the qdbus call failed or com.nokia.mce does not exist in the system.

Reply:
Since it's now using mce exclusively I changed it to "$path/mce" and the file shouldn't get created if calling mce returns an error.